openbb_platform/CONTRIBUTING.md

```diff
@@ -72,28 +186,28 @@ After you've defined both models, you'll need to define a `Fetcher` class which
 2. `extract_data` - makes the request to the API endpoint and returns the raw data.
 3. `transform_data` - transforms the raw data into the defined data model.
 
-> Note that the `Fetcher` should inherit from the [`Fetcher`](openbb_platform\platform\provider\openbb_provider\abstract\`Fetcher`.py) class, which is a generic class that receives the query parameters and the data model as type parameters.
+> Note that the `Fetcher` should inherit from the [`Fetcher`](platform/provider/openbb_provider/abstract/fetcher.py) class, which is a generic class that receives the query parameters and the data model as type parameters.
 
-After finalizing your models you need to make them visible to the Platform. This is done by adding the `Fetcher` to the `__init__.py` file of the `<your_package_name>/<your_module_name>` folder as part of the [`Provider`](openbb_platform\platform\provider\openbb_provider\abstract\provider.py).
+After finalizing your models, you need to make them visible to the Openbb Platform. This is done by adding the `Fetcher` to the `__init__.py` file of the `<your_package_name>/<your_module_name>` folder as part of the [`Provider`](platform/provider/openbb_provider/abstract/provider.py).
 
-Any command using the `Fetcher` class you've just defined, will be calling the `transform_query`, `extract_data` and `transform_data` methods under the hood in order to get the data and output it do the end user.
+Any command, using the `Fetcher` class you've just defined, will be calling the `transform_query`, `extract_data` and `transform_data` methods under the hood in order to get the data and output it do the end user.
```

Comment:
Aren't all data collection endpoints required to be a Fetcher class? We should be very clear about this.

Reply:
Good point, we should add more clarity here. Not all of them need to use a Fetcher class. I do imagine some people just grabbing the data inside the router and doing logic there if they wish to. Especially for alternative data or things that they don't want to go an extra step for. The user will always have the freedom but we have the freedom to propose the expected steps and implementations that we approve and use.

Reply:
If they are "just grabbing some data inside the router", this should imply that the Fetcher already exists.  Otherwise, it circumvents all the other procedures.  If I knew we weren't seriously going to enforce them, I would have done that ages ago to get around the I/O constraints of passing data internally.